maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/CCOutcomeController.java

```diff
@@ -106,11 +106,33 @@ public ResponseEntity<RepOrderCCOutcomeDTO> update(@Parameter(description = "Rep
     public ResponseEntity<List<RepOrderCCOutcomeDTO>> findByRepId(HttpServletRequest request, @PathVariable int repId) {
         log.info("Find RepOrder CC Outcome Request Received");
         validator.validate(repId);
-        if (request.getMethod().equals(RequestMethod.HEAD.name())) {
-            HttpHeaders responseHeaders = new HttpHeaders();
-            responseHeaders.setContentLength(service.findByRepId(repId).size());
-            return ResponseEntity.ok().headers(responseHeaders).build();
-        }
         return ResponseEntity.ok(service.findByRepId(repId));
     }
+
+    @RequestMapping(value = "/reporder/{repId}",
+            method = {RequestMethod.HEAD},
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    @Operation(description = "Retrieve a RepOrder CCOutCome size in the header")
```

Comment:
(Not as part of this change) Going forwards we should try and see if we can reduce a lot of this boiler plate annotation code which is pretty much identical across all endpoints and controllers, maybe an approach like the below but in a more reusable/flexible way, maybe one annotation per return code?

https://github.com/ministryofjustice/laa-maat-court-data-api/blob/d8286767c4347df1e81fe6c99bcf5ad07de22379/maat-court-data-api/src/main/java/gov/uk/courtdata/eform/controller/StandardApiResponseCodes.java

Reply:
Yep, agree. Also, I'm not 100% confident we actually guarantee these error responses!